src/cpip-pack

```diff
@@ -74,7 +74,7 @@ install_compilers()
 
         # install compilers
         local pkgs=("$cc_pkg" "$cxx_pkg" "$gfortran_pkg" "$swig_pkg" "$ld_impl" "$pkg_config")
-        local pkg_diff="$(conda install --override-channels -c anaconda ${pkgs[@]} --no-update-deps --json --quiet)"
+        local pkg_diff="$(conda install --override-channels -c anaconda -c conda-forge ${pkgs[@]} --no-update-deps --json --quiet)"
```

Comment:
Not a huge fan of having to add `conda-forge`, but understand we have no other choice in order to obtain the specific version of swig.

Reply:
What we will do, is bring `swig` to `4.0.2` and keep `anaconda` channel only.